.gitignore

```diff
@@ -11,7 +11,6 @@ node_modules
 *.log
 /dist/*
 /_sqldata/*
-temp
```

Comment:
Could we keep ignoring the /temp directory, or is there a reason you want to track it in git?

Reply:
I think this was a mistake. Sorry.